### ISSUE TITLE
Docs: Feedback for Near Cache

### ISSUE THREAD

**Issue by naadhira** [hazelcast/hz-docs#319]
Hi, I have some feedback about [this page](https://github.com/hazelcast/hz-docs/edit/main/docs/modules/performance/pages/near-cache.adoc)

I don't think this feature is in the right place in the docs. Near Cache moves a data store to the client side and really has nothing to do with queries. In fact, the docs say right there that Near Cache is better for frequent reads, not for queries.

**Comment by Serdaro** [comment]
Thank you @naadhira , I think you are right. When migrated content from IMDG docs to Platform docs, the content in question was moved from "Performance" chapter of IMDG docs to "Improving Query Performance" of Platform docs.

I've realized 95% of the Near Cache content is duplicated in https://docs.hazelcast.com/hazelcast/5.4-snapshot/performance/near-cache#latest-banner and https://docs.hazelcast.com/hazelcast/5.4-snapshot/cluster-performance/best-practices#near-cache.

I am suggesting to simplify the issue by putting the whole content under https://docs.hazelcast.com/hazelcast/5.4-snapshot/cluster-performance/best-practices#near-cache , and give references from Map and JCache sections.

Sounds good?
